src/violations.rs

```diff
@@ -3218,6 +3218,34 @@ impl AlwaysAutofixableViolation for FunctoolsCache {
     }
 }
 
+define_violation!(
```

Comment:
Violations have moved to the specific module, might as well move these directly into `pyupgrade`

Reply:
I love this refactor! @charliermarsh, say the word and I will do it for the rest of the modules in pyupgrade.